packages/textfield/index.js

```diff
@@ -52,20 +49,19 @@ class WarpTextField extends WarpElement {
   }
 
   get _inputStyles() {
-    return fclasses({
-      [input.default]: true,
-      [input.invalid]: this.invalid,
-      [input.disabled]: this.disabled,
-      [input.readOnly]: this.readOnly,
+    return classes({
+      [input.base]: true,
+      [input.default]: !this.invalid && !this.disabled && !this.readOnly,
```

Comment:
The way `fclasses` util was handling arguments made that classes strings with `undefined` value would override classes with truthy value. Here e.g. `'border-1': true` (`[input.default]: !this.invalid ...`) would be overridden by `border-1: undefined` (`[input.default]: this.invalid ...`). The resulting style would be like this:
![Screenshot 2024-05-03 at 12 32 34](https://github.com/warp-ds/elements/assets/41303231/8ffce92b-7b47-445e-a420-c748dfdae422)